modules/imgcodecs/test/test_grfmt.cpp

```diff
@@ -208,6 +216,88 @@ INSTANTIATE_TEST_CASE_P(All, Imgcodecs_ExtSize,
                             testing::ValuesIn(all_exts),
                             testing::ValuesIn(all_sizes())));
 
+
+
+
+typedef tuple<Size, int> Size_thresh;
+typedef testing::TestWithParam<Size_thresh> Imgcodecs_Image_pbm_thresh;
+
+
+TEST_P(Imgcodecs_Image_pbm_thresh, read_write_pbm)
+{
+    const String ext = "pbm";
+    const Size size = get<0>(GetParam());
+    const int thresh = get<1>(GetParam());
+    const string full_name = cv::tempfile(ext.c_str());
+    const string _name = cvtest::TS::ptr()->get_data_path() + "readwrite/marbles.pbm";
```

Comment:
Test data is missing.
You should put testdata into opencv_extra repository with same branch name (`write_pbm`) and create PR